menas/src/main/resources/server.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This whole file can be condensed, remove any unneeded lines, I would remove anything extra other than the licence